defaults.h

```diff
@@ -128,16 +128,16 @@
 #define MIN_RANDOM_FACTOR (1.0 / 2.0)
 #define MAX_RANDOM_FACTOR (3.0 / 2.0)
 
-/* MAX and MIN (RFC4861), Mobile IPv6 extensions will override if in use */
+/* MAX and MIN (RFC4861 and RFC8316), Mobile IPv6 extensions will override if in use */
 
 #define MIN_MaxRtrAdvInterval 4
-#define MAX_MaxRtrAdvInterval 1800
+#define MAX_MaxRtrAdvInterval 65535 /* Defined in RFC8316 Section 4 */
```

Comment:
Here a fanboy of "source code should be readable as a good book", hence I suggest
```C
#define MAX_MaxRtrAdvInterval 65535 /* as RFC8316 Section 4 says */
```

Reply:
I've just put the same comment that you suggested for the previous line.